LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (C) 2014-present Dogfessional
+Copyright (C) 2014-present Formidable
```

Comment:
OPTIONAL: We typically preserve prior (C) to the author, so would be something like:

```
Copyright (C) 2014-2020 Emil Hartz
Copyright (C) 2020 Formidable Labs, Inc.
```

... and I'm adding in a real name unless Dogfessional is a real company with assigned ownership (and if so, that's amazing...)